trade-finance-manager-ui/test-helpers/mocks/entra-id.config.mock.builder.ts

```diff
@@ -0,0 +1,34 @@
+import { EntraIdConfig } from '../../server/configs/entra-id.config';
+import { BaseMockBuilder } from './mock-builder.mock.builder';
+
+export class EntraIdConfigMockBuilder extends BaseMockBuilder<EntraIdConfig> {
+  constructor() {
+    super({
+      defaultInstance: {
+        msalAppConfig: {
+          auth: {
+            clientId: 'a-client-id',
+            authority: undefined,
+            clientSecret: undefined,
+            clientAssertion: undefined,
+            clientCertificate: undefined,
+            knownAuthorities: undefined,
+            cloudDiscoveryMetadata: undefined,
+            authorityMetadata: undefined,
+            clientCapabilities: undefined,
+            protocolMode: undefined,
+            azureCloudOptions: undefined,
+            skipAuthorityMetadataCache: undefined,
+          },
+          broker: undefined,
+          cache: undefined,
+          system: undefined,
+          telemetry: undefined,
```

Comment:
Do we need to define all of these fields with undefined values?

Reply:
Great spot, i played around and reduced this. thanks!

Reply:
I forgot to sync! Sorry -- should be fixed